pkg/resources/mutation/cluster/provisioningcluster.go

```diff
@@ -29,6 +30,9 @@ func (m *mutator) Admit(response *webhook.Response, request *webhook.Request) er
 	if err != nil {
 		return err
 	}
-
-	return mutation.SetCreatorIDAnnotation(request, response, cluster, cluster.DeepCopy())
+	if err := mutation.SetCreatorIDAnnotation(request, response, request.Object, cluster.DeepCopy()); err != nil {
+		return fmt.Errorf("failed to set creatorIDAnnotation: %w", err)
+	}
+	response.Allowed = true
```

Comment:
wonder how this could have worked before :thinking:

Reply:
Before, SetCreatorIDAnnotation would also set `response.Allowed`, but this was moved out since setting an annotation does not guarantee that the request is valid for anyone else using the function.